rfd/0052-passwordless.md

```diff
@@ -436,6 +436,104 @@ attestation lists, in order to avoid configurations that are impossible to
 satisfy. The presence of an admin-defined attestation list is also interpreted
 as a decision, by the admin, to take control over such configurations.
 
+### Cluster settings
+
+As explained in the [authentication](#authentication) section, the decision to
+perform a passwordless login is commonly made by the end user. This is because
+Teleport has no information about the user prior to login. In order to provide a
+more seamless passwordless experience, admins may switch the default login
+method of a cluster to passwordless.
+
+Fallback methods are provided for users who may lack passwordless credentials:
+namely `tsh --auth` for CLI and UI options for the Web UI (not discussed in this
+design).
+
+Any cluster capable of WebAuthn is, by default, capable of passwordless.
+Teleport also offers a toggle to disable passwordless, in case admins so desire.
+
+teleport.yaml:
+
+```yaml
+auth_service:
+  authentication:
+    type: local
+    second_factor: on
+    webauthn:
+      rp_id: "example.com"
+
+    # Explicitly enable/disable passwordless.
+    # Defaults to "true".
+    passwordless: true
+
+    # Sets passwordless as the default connector.
+    connector_name: passwordless
```

Comment:
FYI, @hatched just added some code to improve the error message for unknown connector names. It will need to be updated to account for `passwordless` as a special case (like `local`).